tidy3d/components/simulation.py

```diff
@@ -989,6 +991,25 @@ def _validate_monitor_size(self) -> None:
                 f"a maximum of {MAX_SIMULATION_DATA_SIZE_GB:.2f}GB are allowed."
             )
 
+    @cached_property
+    def monitors_data_size(self) -> typing.Dict:
+        """calculate simulation monitors storage size"""
```

Comment:
Maybe it would be good to be a little more specific. 
```
"""Dictionary mapping monitor names to their estimated storage in bytes."""
```
or something

Reply:
changed.